ref/general/packaging-deploying.adoc

```diff
@@ -14,79 +14,69 @@
 = Package and deploy applications with Open Liberty
 
 You can choose among different options for packaging, containerizing, and deploying applications with Open Liberty.
-However, the simplest workflow for developing your cloud-native applications is to build applications with Maven or Gradle, containerize server packages with Docker, and deploy to the cloud with Kubernetes or OpenShift.
+However, the simplest workflow for developing your cloud-native applications is to build applications with Maven or Gradle, containerize with Docker, and deploy to the cloud with Kubernetes or OpenShift.
```

Comment:
I am not keen on starting this sentence with however. It feels kind of negative. Also the prior sentence references containerizing, but the options being sort of excluded here include not containerizing.